### ISSUE TITLE
Compendium crashes on resizing of screen in lates build from M1

### ISSUE THREAD

**Issue by SebastianBoell** [CompendiumNG/CompendiumNG#74]
I just noticed that the latest build from M1 crashes when the window is resized. I tried to change window settings under /System/resources/Format.properties Settings are correctly recognized if I change the size there but Compendium still keeps crashing. The crash is pretty major as trying to close the window or usint Alt F4 does not work.

**Comment by maiklos** [comment]
I was able to reproduce it. It seems it has something to do with toolbars. Once you change the width of the main window so the toolbar doesn't fit the width anymore it just hangs.

workarounds: minimize toolbars

**Comment by maiklos** [comment]
fixed in 11e28 - fix merged into M1 branch

method that tries to collapse some toolbar in order to save space was cycling

**Comment by SebastianBoell** [comment]
You are spot on it has to do with the toolbars.  Tested and it applies to horizontally as well as vertically arranged toolbars. Workaround with minimizing toolbars works fine.

Unfortunately the bug fix does not seem to be working. It still crashes on my machine if resizing into one of the toolbars. 

I checked against 2.0 Beta there minimizing the toolbars when resizing the screen works correctly.

I will reopen this but. I leave it as critical as it would need to be fixed for a first release.
